PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -94,6 +94,8 @@
         # RUN
         'keep LHERunInfoProduct_*_*_*',
         'keep GenRunInfoProduct_*_*_*',
+        'keep *_genParticles_xyz0_*',
+        'keep *_genParticles_t0_*',
```

Comment:
this propagates to all of miniAOD.
@gpetruc @arizzi 
Is it OK? or should we restrict to the timing eras?

Reply:
The xyz0 is useful in general.

The t0 could in principle be restricted, but it's only one float per event, and might be useful already for exotic particle searches.

Reply:
apologies for my ignorance, what does xyz0 contain?

Reply:
The position of the hard interaction.

Reply:
ok I imagined so, well we usually get it from one of the very first prunedGenParticles but I'd say this is a safer  way and it costs nothing on the total budget